packages/salesforcedx-vscode-lwc/package.nls.json

```diff
@@ -1,8 +1,8 @@
 {
-  "force_lightning_lwc_start_text": "SFDX: Start Local Development Server",
-  "force_lightning_lwc_stop_text": "SFDX: Stop Local Development Server",
-  "force_lightning_lwc_open_text": "SFDX: Open Local Development Server",
-  "force_lightning_lwc_preview_text": "SFDX: Preview Component Locally",
+  "force_lightning_lwc_start_text": "SFDX: Start Component Preview",
```

Comment:
@LisandroFernandezSF 
Per the latest feedback on the work item (W-8247907):

SFDX: Start LWC Code Preview Server
SFDX: Stop LWC Code Preview Server
SFDX: Open Code Preview Home

FYI: @smaddox-sf